pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -236,6 +236,7 @@ type ClusterSpec struct {
 	Authorization *AuthorizationSpec `json:"authorization,omitempty"`
 
 	// Tags for AWS instance groups
+	NodeLabels map[string]string `json:"nodeLabels,omitempty"`
```

Comment:
Why are we adding an object since we already have an object like this in instance groups?  I am confused.